containers/groovy-examples/.gitignore

```diff
@@ -0,0 +1,5 @@
+notebooks/notebooks/*
+!notebooks/notebooks/01-example.py
+
+data/*
+!data/.ignore
```

Comment:
We may want to employ a strategy at our root gitignore to *keep* any files named `.gitkeep`; I think I've seen this used elsewhere. That way we only need the logic in one .gitignore file.

The one problem we found with that is then there is a `.gitkeep` file in notebooks UI. There are various ways we could solve this problem, ranging from the client knowing to ignore that file, or from updated the webdav server to ignore it.